app/presenters/race_edition_presenter.rb

```diff
@@ -16,10 +15,33 @@ def category_size_map
     categories.map { |category| [category.name, grouped_race_entries[category.name]&.size || 0] }
   end
 
+  def edition_count
+    return nil unless year.present?
+
+    year - 2004
+  end
+
+  def short_description
+    case race&.name
+    when "Rattlesnake Ramble Trail Race"
```

Comment:
Is this formatting correct? Shouldn't each "when" be indented?

Reply:
Standard Ruby style is to indent `when` to the same level as `case`. https://github.com/rubocop-hq/ruby-style-guide#indent-when-to-case